keycloak/keycloak-admin-ui
# Create profile

## Motivation
1. Add deleting client profile
2. Add cypress tests for realm settings client profiles tab tests

![Screenshot 2021-09-23 at 19 57 41](https://user-images.githubusercontent.com/4890675/134644971-5932a2a1-cd44-4331-a8b6-80965534f10b.png)


3. Add creating new client profile - wip (adding executors not fully implemented)

## Verification Steps
1. Go to /realm-settings/clientPolicies/new-client-profile and create a new client (add executors is wip/ not working yet)
2. Go to /realm-settings/clientPolicies, select <more icon> for the created profile, select delete and confirm profile deletion. Verify that the delete profile is not listed in the profiles table.
3. Run realm settings client profile tests.
